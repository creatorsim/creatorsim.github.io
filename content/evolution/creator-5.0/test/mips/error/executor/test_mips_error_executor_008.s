
#
# Creator (https://creatorsim.github.io/creator/)
#

.data
	a: .space 4

.text
main:
	
	la $t0, a
    li $t1, 10
	lw $t1, 1($t0)