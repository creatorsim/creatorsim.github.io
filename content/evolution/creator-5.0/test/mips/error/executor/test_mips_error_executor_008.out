
The memory must be align

 Error found.
 The program has finished with errors
cr[PC]:0x14; ir[1,at]:0x200000; ir[8,t0]:0x200000; keyboard[0x0]:''; display[0x0]:''; 

